### ISSUE TITLE
Elasticsearch-php produces lots of warnings with php 7.4 preloading enabled.

### ISSUE THREAD

**Issue by hoebelix** [elastic/elasticsearch-php#1042]
### Summary of problem or feature request

I use elasticsearch-php within a symfony project (symfony 4.4). Elasticsearch-php produces lots of warnings with php 7.4 preloading enabled.

Example log line:
NOTICE: PHP message: PHP Warning:  Cannot declare class \Elasticsearch\Endpoints\Cluster\Nodes\HotThreads, because the name is already in use in /code/xx/vendor/elasticsearch/elasticsearch/src/autoload.php on line 11

This line is repeated for every class_alias defined in the mentioned autoload.php

### System details

- Operating System: Alpine Linux
- PHP Version: 7.4.6
- ES-PHP client version: 7.7.0
- Elasticsearch version: 7.7

**Comment by adri** [comment]
I'm seeing the same issue with PHP 7.4.7. 

```
PHP 7.4.7 (cli) (built: Jun 11 2020 18:41:17) ( NTS )
Copyright (c) The PHP Group
Zend Engine v3.4.0, Copyright (c) Zend Technologies
    with Zend OPcache v7.4.7, Copyright (c), by Zend Technologies
```

**Comment by ezimuel** [comment]
@hoebelix, @adri we are working on #1051 but I would like to understand the preload configuration that  you are using. Thanks!

**Comment by hoebelix** [comment]
I just use the default symfony 4.4 preload configuration:

opcache.preload=/var/www/var/cache/prod/srcApp_KernelProdContainer.preload.php
opcache.preload_user=www-data

**Comment by adri** [comment]
Same for me, default symfony 4.4

**Comment by ezimuel** [comment]
@hoebelix, @adri thanks for the information. The issue has been fixed in #1051. I'll release it soon in 7.9.

**Comment by adri** [comment]
Awesome thanks!
